airflow/contrib/example_dags/example_gcp_compute.py

```diff
@@ -21,15 +21,15 @@
 Example Airflow DAG that starts, stops and sets the machine type of a Google Compute
 Engine instance.
 
-This DAG relies on the following Airflow variables
-https://airflow.apache.org/concepts.html#variables
+This DAG relies on the following OS environment variables
```

Comment:
Why do we want to rely on environment variables?

Reply:
This is because of the dicsussion here: https://lists.apache.org/list.html?dev@airflow.apache.org:lte=1M:variables . We are moving away from airflow variables in DAGs (at least in the top-level) because it might show an example of anti-pattern - using airflow variables in DAGs at top level causes a lot of opened coneections to the DB every 5 seconds. We also change a way we run the DAGs to the unit test-driven approach (see below) and it makes it easier to set environment variables in IDE to run them very easily.